components/brave_new_tab_ui/components/default/exchange/style.ts

```diff
@@ -0,0 +1,174 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+interface StyleProps {
+  itemsShowing?: boolean
+  isSmall?: boolean
+  isInTab?: boolean
+}
+
+export const StyledTitleTab = styled<StyleProps, 'div'>('div')`
+  color: #fff;
+  cursor: pointer;
+  padding: 10px ${p => p.isInTab ? 13 : 25}px;
+  margin-bottom: -3px;
+  border-radius: 6px 6px 0 0;
+  background-image: linear-gradient(270deg, rgba(33,37,41,0.16) 0%, #212529 100%);
```

Comment:
I think backdrop filter will get a nicer glass-like effect that also the design seems to call for

Reply:
added